RadarSDK/RadarAPIClient.m

```diff
@@ -244,106 +333,66 @@ - (void)trackWithLocation:(CLLocation *_Nonnull)location
 
     NSDictionary *headers = [RadarAPIClient headersWithPublishableKey:publishableKey];
 
-    [self.apiHelper requestWithMethod:@"POST"
-                                  url:url
-                              headers:headers
-                               params:params
-                                sleep:YES
-                           logPayload:YES
-                    completionHandler:^(RadarStatus status, NSDictionary *_Nullable res) {
-                        if (status != RadarStatusSuccess || !res) {
-                            if (options.replay == RadarTrackingOptionsReplayStops && stopped &&
-                                !(source == RadarLocationSourceForegroundLocation || source == RadarLocationSourceManualLocation)) {
-                                [RadarState setLastFailedStoppedLocation:location];
-                            }
-
-                            [[RadarDelegateHolder sharedInstance] didFailWithStatus:status];
-
-                            return completionHandler(status, nil, nil, nil, nil, nil);
-                        }
-
-                        [Radar flushLogs];
-                        [RadarState setLastFailedStoppedLocation:nil];
-
-                        RadarMeta *_Nullable meta = [RadarAPIClient parseMeta:res];
+    if (options.replay == RadarTrackingOptionsReplayAll) {
```

Comment:
Think we probably want: ` && replay buffer size > 0`, so regular location updates would still use `/track`

Reply:
The thinking of omitting that condition was that if we do this, we'll need to add conditional logic about adding failed calls to the buffer in the `/track` callback.

But it makes sense to get as many calls as possible the tried & tested `/track` codepath, so I'll go ahead and make this change.